setup.py

```diff
@@ -13,27 +13,23 @@
     'Topic :: Internet :: WWW/HTTP :: Dynamic Content',
     'Topic :: Software Development',
     'Topic :: Software Development :: Libraries :: Application Frameworks',
-    'Programming Language :: Python :: 3.4',
-    'Programming Language :: Python :: 3.5',
     'Programming Language :: Python :: 3.6',
```

Comment:
3.6 isn't being tested against and is EOL, so could we drop it in this PR?

Reply:
The one issue that this creates is that we have to make any changes twice as it is being used with Py3.6 still. I had kept it in the test suite but this was kind of a middle ground to help the changes come in earlier and not have to maintain two different codebases. It's tested on py3.6 locally to ensure compatibility. When you remove this flag you can't install the package on 3.6, pip refuses. :-(

Reply:
That's fair enough. If it's still necessary, then all is well 👍